data/cards-expansion_set-shatteredAlliances.cfg

```diff
@@ -163,4 +180,74 @@
 		
 		}
 	},
+	
+	"I am Legion": {
+		name: "I am Legion",
+		rarity: 2,
+		set: "shatteredAlliances",
+		type: "creature",
+		school: "@eval BLOOD",
+		flavor_text: '"My name is Legion", he replied, "...for we are many."', //how to make parts of it italics?  
```

Comment:
Related to: https://github.com/davewx7/citadel/blob/57344282748ecbc42e41c6288915fdaa834dff64/data/cards-aether.cfg#L803